uiowa/uiowa01
# Feature: Set page title and breadcrumbs block visibility when installing content types with LB enabled

This feature branch addresses issue #137.
 
### How to test

1. Run collegiate_basic install.
2. The install should break due to the preprocess function that isn't working.
3. Work with the preprocess function on line 77 of uiowa_lb_blocks.module.
4. Once the function is working, where should we move the function?
